src/NuGetPackageVerifier/Rules/AssemblyHasReleaseConfigurationRule.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace NuGetPackageVerifier.Rules
 {
-    public class AssemblyHasCorrectBuildConfigurationRule : AssemblyHasAttributeRuleBase
+    public class AssemblyHasReleaseConfigurationRule : AssemblyHasAttributeRuleBase
```

Comment:
We had a hard time figuring out what the name of Release configuration rule was. Having `Release` in the name would have helped